app.css

```diff
@@ -1,3 +1,210 @@
 * {
   box-sizing: border-box;
+  font-family: sans-serif;
+}
+
+.banner {
+  max-width: auto;
+  height: 19rem;
+  background-image: url(http://lorempixel.com/400/200);
+}
+
+.navbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 auto;
+  max-width: 55rem;
+  padding-top: 1rem;
+}
+
+.links {
+  display: flex;
+}
+.link__item {
+  margin: 0 auto;
+  width: 5rem;
+}
+
+.navbar__logo {
+
+}
+
+.hero {
+  text-align: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.about {
+
+}
+
+.search {
+
+}
+
+.title {
+  font-size: 60px;
+  color: white;
+}
+
+.subtitle {
+}
+
+/* Left Article */
+.left-article {
+  display: flex;
+  flex-direction: column;
+  margin-left: 5rem;
+  justify-content: space-between;
+  max-width: 70rem;
+  margin-top: 3rem;
+  height: 25rem;
+}
+
+.article__heading {
+  font-weight: bolder;
+  font-size: 37px;
+  max-width: 50%;
+  color: #030a21;
+}
+
+.article__info {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  max-width: 16rem;
+}
+
+.article__author-avatar {
+  border-radius: 50%;
+}
+
+.article__publish-date {
+  color: #a19ec1;
+  font-size: 12px;
+}
+
+.author__name-left {
+  font-size: 13px;
+}
+
+.article__text {
+  font-size: 14px;
+  max-width: 40%;
```

Comment:
This is moving a lot of the column logic into the individual text... That's doing more work than needed. Instead make the whole left column group a particular width, then everything else falls in line.